home/models.py

```diff
@@ -15,13 +15,22 @@
     StreamFieldPanel,
 )
 from wagtail.contrib.settings.models import BaseSetting, register_setting
+from wagtail.search import index
 from django.utils.translation import gettext_lazy as _
 
 
 class HomePage(Page):
-    subpage_types = [
-        "SectionPage",
-    ]
+    subpage_types = ["SectionPage", "ArticleTagIndexPage"]
+
+    def get_context(self, request):
+        # Update context to seperate sectionpages and tag index
+        context = super().get_context(request)
+        sections = SectionPage.objects.child_of(self).live()
+        context["sections"] = sections
+
+        tags = [tag.tag.name for tag in ArticlePageTag.objects.all()]
```

Comment:
One suggestion I would make would be to consistently use the tag's slug rather than the name when referencing it in URLs - that will make them a bit more human-readable and avoid any issues with escaping if there are unexpected characters in them such as spaces. For this line I'd go for:

```suggestion
        tags = [tag.tag for tag in ArticlePageTag.objects.all()]
```

(so that it's returning a set of Tag objects, rather than just names)
and then in the home_page.html template (and similarly on article_page.html):

    <p><a href="{% slugurl 'tags' %}?tag={{ tag.slug }}">#{{ tag.name }}</a></p>

Then the lookup on `ArticleTagIndexPage` would be

    articlepages = ArticlePage.objects.filter(tags__slug=tag)

Reply:
We've removed these links since they're not wanted anymore.